include/rdma/fi_ext.h

```diff
@@ -203,6 +260,20 @@ struct fi_peer_srx_context {
 	struct fid_peer_srx *srx;
 };
 
+
+/*
+ * Peer transfers
+ */
+struct fi_peer_transfer_context {
+	size_t size;
+	struct fi_info *info;
+	struct fid_domain *domain;
+	struct fid_ep *ep;
+	struct fid_cq *cq;
+	struct fid_peer_av *av;
```

Comment:
The AV may not be needed either.  The main provider could just bind the EP with the AV.  This would basically be an optimization.  If we go this route, the peer CQ(s) could be added too.